lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -152,9 +152,9 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
                 preventCorsPreflight: true
             },
             loggerOptions: {
-                loggerCallback: this.config.system.loggerOptions.loggerCallback,
-                piiLoggingEnabled: this.config.system.loggerOptions.piiLoggingEnabled,
-                logLevel: this.config.system.loggerOptions.logLevel,
+                loggerCallback: this.config.system?.loggerOptions?.loggerCallback,
```

Comment:
These should all be defined by this point in the flow (with defaults assigned if no logger was provided by the developer), so these '?' should not be needed

Reply:
have refactored and removed '?' , this PR is to cover when loggerOptions explicitly set to undefined/ void 0 to prevent breaking. when no logger is provided, I agree this is not needed at all.

Reply:
Addressed & resolving